### PR TITLE
Make button draggable

### PR DESCRIPTION
**Pull request by mloginov**
https://codio.myjetbrains.com/youtrack/issue/codio-15044/Make-guides-linter-button-draggable

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -260,12 +262,55 @@
...
+    }
+
+    const mouseMoveHandler = function (e) {
+      drag = true
+      // How far the mouse has been moved
+      const dx = e.clientX - x
+      const dy = e.clientY - y
+
+      // Set the position of element
+      button.style.top = `${button.offsetTop + dy}px`
```
Comment (destitutus) [inline index.js:293]: could the position be stored in local storage and fetched after restart?

Comment (mloginov) [reply index.js:302]: discussed with Veronika - not needed for now. Lets wait for feedback

Comment (mloginov) [reply index.js]: done

File: index.js [hunk 2]
```diff
@@ -250,7 +253,11 @@
       button.id = LINTER_BUTTON_ID
       button.innerHTML = 'Check guides'
       button.type = 'button'
-      button.onclick = async () => {
+      const data = JSON.parse(localStorage.getItem(CODIO_GUIDES_LINTER))
```
[the remark this answers is not in the document]
Comment (mloginov) [reply index.js]: fixed

**Review by destitutus — APPROVED** [review]

**Pull request merged**